client/components/payment-methods-checkboxes/payment-method-checkbox.scss

```diff
@@ -72,6 +78,14 @@
 		text-transform: initial;
 	}
 
+	&__required-label {
+		display: block;
+		font-size: 11px;
+		line-height: 1em;
+		color: $gray-700;
+		text-transform: initial;
```

Comment:
As it's a child of the parent label `text-transform: initial` is alrady applied.

Reply:
Good catch.